### ISSUE TITLE
Two albedo in a simulation

### ISSUE THREAD

**Issue by greenyangca** [NREL/bifacial_radiance#389]
I am just learning bifacial_radiance, and need to do analysis for two different albedo on ground. The simulation is for tracker, the area under tracker and aisle between tracker have different albedo. Is it possible to do analysis with bifacial_radiance?

**Comment by shirubana** [comment]
Yes, [Tutorial 14](https://github.com/NREL/bifacial_radiance/blob/main/docs/tutorials/14%20-%20Advanced%20topics%20-%20Cement%20Pavers%20albedo%20example.ipynb) has a good exxample for this, or also [tutorial 11](https://github.com/NREL/bifacial_radiance/blob/main/docs/tutorials/11%20-%20Advanced%20topics%20-%20AgriPV%20Systems.ipynb)
